src/handlers/psp.ts

```diff
@@ -25,24 +27,32 @@ import {
   SponsorWalletWithSubscriptions,
   State,
 } from '../types';
-import { retryGo } from '../utils';
+import { DEFAULT_RETRY_TIMEOUT_MS } from '../constants';
 import { Subscription } from '../validator';
 import { shortenAddress } from '../wallet';
 
 export const handler = async (_event: any = {}): Promise<any> => {
   const startedAt = new Date();
 
-  const airnodeConfig = loadAirnodeConfig();
+  const airnodeConfig = goSync(loadAirnodeConfig);
+  if (!airnodeConfig.success) {
+    utils.logger.error(airnodeConfig.error.message);
+    throw airnodeConfig.error;
+  }
```

Comment:
> I would prefer not to log things in this file but maybe we can try/catch and return [logs, config] or [error, config] something like that and the caller would do the logging. This is just an idea

Since we don't want to log anything in **config.ts**, how about using the `goSync` function from **promise-utils** as above?

This way we don't need to modify the loadConfig functions to return tuples. :thinking: 

What do you think? @acenolaza